cypress/integration/login.js

```diff
@@ -19,6 +19,7 @@ describe('Login functionality', function() {
         cy.get('.calendar').should('be.visible')
         cy.get('.tile.form').should('be.visible')
         cy.get('.user-info__title').contains('Demo User')
+        cy.get('[data-test="timesheets-current-day"]').contains(15)
```

Comment:
This test will fail tomorrow, maybe we can use something better? e.g. a hint if you'll get stuck :) https://www.w3schools.com/jsref/jsref_getday.asp

Reply:
https://www.w3schools.com/jsref/jsref_getdate.asp